game ver.commit 1.js

```diff
@@ -73,6 +73,21 @@ function drawBricks() { //벽돌 좌표 지정 + 그리기
             } } }
 }
 
+var score=0;
+function callScore(jewel){/*점수함수*/
+    var bonus;

```

Comment:
const, let을 써보는 것도 괜찮을 듯 하네요
```suggestion
let score=0;
function callScore(jewel){/*점수함수*/
    let bonus;

```